CHANGELOG.md

```diff
@@ -2,12 +2,42 @@
 
 ## Unreleased
 
+## 3.13.0 (Feature release)
+
+### New features
+
+#### Add a 'none' option and 'or' divider to checkboxes
+
+You can now add a 'none' option to checkboxes. For example, 'None of the above' or 'None of these options apply to me.'
+
+Use the 'none' option to [allow users to state that none of the checkboxes apply to them](https://design-system.service.gov.uk/components/checkboxes/#add-an-option-for-none-). Otherwise, users would have to work out that they should leave all the checkboxes unticked.
```

Comment:
Is there going to be content in this page on adding an option for none? I don't see any on this link at the moment

Reply:
There will be content on the page for this option, but our content designer is still finalising it. By the time these release notes go out, the content will be visible under the 'Add an option for none' heading.